src/test/scala/com/cognite/sdk/scala/v1/DataModelsTest.scala

```diff
@@ -8,6 +8,8 @@ import com.cognite.sdk.scala.common.RetryWhile
 
 import java.util.UUID
 import scala.collection.immutable.Seq
+import com.cognite.sdk.scala.v1.DataModelType.Edge
```

Comment:
Revert all the changes in this file from your merge commit, and it should compile again 😀

Reply:
Ahaa, i renamed it to EdgeType, is it wrong?

Reply:
I mean now the test works hehe

Reply:
i think I did the right thing without reading your comment, so we are fine hehe :D